code/game/objects/items/twohanded.dm

```diff
@@ -298,6 +300,45 @@
 	total_mass = 0.4 //Survival flashlights typically weigh around 5 ounces.
 	var/total_mass_on = 3.4
 
+/datum/block_parry_data/dual_esword
+	block_damage_absorption = 5
+	block_damage_multiplier = 0.3
+	block_start_delay = 0		// instantaneous block
+	block_stamina_cost_per_second = 6
+	block_stamina_efficiency = 2
+	// slowdown
+	block_slowdown = 1
+	// no attacking while blocking
+	block_lock_attacking = TRUE
+
+	parry_time_windup = 0
+	parry_time_active = 8
+	parry_time_spindown = 0
+	// we want to signal to players the most dangerous phase, the time when automatic counterattack is a thing.
+	parry_time_windup_visual_override = 1
+	parry_time_active_visual_override = 3
+	parry_time_spindown_visual_override = 4
+	parry_flags = PARRY_DEFAULT_HANDLE_FEEDBACK		// esword users can attack while parrying.
+	parry_time_perfect = 2		// first ds isn't perfect
+	parry_time_perfect_leeway = 1
+	parry_imperfect_falloff_percent = 20
+	parry_efficiency_to_counterattack = 100
+	parry_efficiency_considered_successful = 25		// VERY generous
+	parry_efficiency_perfect = 90
+	parry_failed_stagger_duration = 3 SECONDS
+	parry_failed_clickcd_duration = CLICK_CD_MELEE
+	parry_cooldown = 3 SECONDS
+
+// KEV, WHY AREN'T YOU JUST PUTTING IT IN THE DEFINITION?!?!?!?!
+// I'LL TELL YOU WHY, BECAUSE BYOND DOESN'T CONSIDER "[(1<<0)]" A "CONSTANT EXPRESSION"
+// WHAT EVEN IS MORE CONSTANT THAN 1 BITSHIFTED TO THE LEFT BY 0 PLACES AS A STRING?
+/datum/block_parry_data/dual_esword/New()
+	// more efficient vs projectiles
+	block_stamina_efficiency_override = list(
+		"[ATTACK_TYPE_PROJECTILE]" = 4
```

Comment:
someone tell me if i should just bite the bullet and add defines for text versions of attack types so we don't have to start the practice of list defining in New()
because both seem horrible but that just might be less horrible lol